srv/salt/ceph/ganesha/pool/default.sls

```diff
@@ -0,0 +1,44 @@
+
+ganesha_config pool:
+  cmd.run:
+    - name: "ceph osd pool create ganesha_config 1 1"
+    - unless: "ceph osd pool ls | grep -q ganesha_config"
```

Comment:
I would suggest also adding application to the new pool to get rid of the health warning after migration:
```
  cluster:
    id:     3827ca60-d13b-4853-8e2b-1d95915bd850
    health: HEALTH_WARN
            application not enabled on 1 pool(s)
```

Maybe `nfs` application? The command is idempotent. 
```
ceph osd pool application enable ganesha_config nfs
```

Reply:
done.